script.js

```diff
@@ -0,0 +1,83 @@
+var Counter = React.createClass({
+    getInitialState: function () {
+        return {
+            counter: 0
+        };
+    },
+
+    increment: function () {
+        this.setState({
+            counter: this.state.counter + 1
+        })
+    },
+
+    decrement: function () {
+        this.setState({
+            counter: this.state.counter - 1
+        })
+    },
+
+    ComponentWillMount(){
```

Comment:
Wszystkie metody cyklu życia zaczynają się od małej litery. W module jak było napisane z dużej to tylko dlatego, że nazwa ta została użyta na początku zdania.